### PR TITLE
default automatic switch to false, re-enable if disabled too

### LINKED ISSUE #362: Settlement demo - allow concurrent UI operations
**Problem**
Disabled UI and "progress bar" is unnecessary and prevents participants from sending more messages while waiting for responses. It also makes things appear slow. Why shouldn't Alice be able to send more credits to Bob while waiting for him to agree to the first one. 

**Solution**
Remove progress bar. Leave UI enabled.

### LINKED ISSUE #374: UI - automatic toggle not defaulting to false
**Detailed Description**
The automatic toggle isn't defaulting to false when two credits are created back to back.

**Actual Behavior**
Create one credit and set it to automatic
Create another, it defaults to automatic
Settlement also defaults to automatic

**Expected Behavior**
Automatic switch should always be unchecked.

**Environment:**
- Java: [e.g. OpenJDK 11.0.4]
- Node: [e.g. v12.9.1-x86]
- OS: [e.g. Ubuntu 18.04]
- Version: [e.g. 1.0.1]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

### PR DESCRIPTION
**Pull request by gregscullard**
**Detailed description**:
See issue description

**Which issue(s) this PR fixes**:
Fixes #374 
Fixes #362 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Changelog updated
- [ ] Readme updated
- [ ] Tests updated

### CODE REVIEW

File: hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/src/main/resources/static/dialogs/credits/settle-now-dialog.js [hunk 1]
```diff
@@ -70,13 +73,15 @@
                     'Content-Type': 'application/json'
                 },
                 body: postBody
-            }).then(function(response) {
+            }).then(async function(response) {
                 if (response.status===200){
                     showSnackBarMessage("Settlement request sent");
+                    location.hash='settlements';
+                    await renderSettlementsPanel(otherUserId);
```
Comment (zoperado) [inline hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/src/main/resources/static/dialogs/credits/settle-now-dialog.js]: why do we need to block-until this block? The modal closes immediately after any action and the snackbar does asynchronously snow when a response is received. I don't suggest to undo.

Comment (gregscullard) [reply hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/src/main/resources/static/dialogs/credits/settle-now-dialog.js]: Unless we wait, when switching to the settlements panel there won't be new data to show. This waits for the pending settlement to be created before switching.

File: hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/src/main/resources/static/dialogs/credits/settle-now-dialog.js [hunk 2]
```diff
@@ -1,15 +1,20 @@
    function openSettleNowDialog(otherUserId){
+	   
+	   const delay = ms => new Promise(res => setTimeout(res, ms));
```
Comment (zoperado) [inline hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/src/main/resources/static/dialogs/credits/settle-now-dialog.js]: [](url)

Comment (gregscullard) [reply hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/src/main/resources/static/dialogs/credits/settle-now-dialog.js]: removed

**Pull request merged**